ipatests/test_integration/test_trust.py

```diff
@@ -463,6 +357,50 @@ def test_user_uid_resolution_in_external_trust_with_shared_secret(self):
         assert re.search(
             testuser_regex, result.stdout_text), result.stdout_text
 
+    def test_override_homedir(self):
+        """
+        https://pagure.io/SSSD/sssd/issue/2474
+        POSIX attributes, such as shell or home directory, should not be overwritten or missing.
+        """
+        tasks.establish_trust_with_ad(self.master, self.ad_domain,
+                                      extra_args=['--range-type', 'ipa-ad-trust'])
```

Comment:
The bugzilla says that to reproduce the original bug you need "IPA-AD trust with POSIX attributes". Here you establish non-POSIX trust. To establish POSIX trust range type should be "ipa-ad-trust-posix"